change/react-native-windows-2020-05-17-09-26-39-MS_CallbackMultiArgs.json

```diff
@@ -1,8 +0,0 @@
-{
-  "type": "prerelease",
-  "comment": "Support 0..n args in native module callbacks",
```

Comment:
Looks like you pulled in some changes from master, but Github doesn't seem to be able to show a diff against the right thing. How did you merge changes?

Reply:
I've had some funny diffs happen with the change files too. My hypothesis is that it is because these files live for some time and are then removed by the bot. If you merge master in between, your branch has the file, but is gone in master.

Reply:
I just did a `git rebase master`...